### DISCUSSION TITLE
Hijacking a container with Concourse 6.7.3 fails with weird error

### DISCUSSION THREAD

**Discussion by erboucherie** [concourse/concourse#6504] [Help & Support] [answered]
@here , we have just started migrating to Concourse **6.7.3** , and we are facing a weird error when trying to hijack a container : 

> fly -t mytarget i -u https://myurl/builds/1.1
> Backend error: Exit status: 500, message: {"Type":"ExecutableNotFoundError","Message":"exec failed: container_linux.go:367: starting container process caused: exec: \"bash\": executable file not found in $PATH","Handle":"","ProcessID":"","Binary":""}
> 

Is it a known bug .
I launch thois command from an Ubuntu server .
Thanks for your answers

**Comment by taylorsilva** [comment, marked as answer]
The error is saying that it can't find "bash" which is the default command hijack uses. From hijack's help text:
```
[hijack command arguments]
  command:                          The command to run in the container (default: bash)
```
Your container must not contain bash. You can specify a different command at the end of your fly command. `sh` is a good fall back.

This issue describes your problem: https://github.com/concourse/concourse/issues/5877

There's a change in how fly will hijack for the next release. It will try `bash` first and then `sh` next. It'll be in the next release, 7.0.0
https://github.com/concourse/concourse/pull/6098

**Reply by erboucherie** [reply]
Thanks for the answer . Indeed launching **sh** works . It's a bit weird that such shell choice changed between Concourse 5 and 6 . I didn't check in the release notes if it was well described . Anyway thanks again for taking my point and for the workaround .
